### PR TITLE
Unlink replica

### PR DESCRIPTION
**Pull request by joshuaj194**

### CODE REVIEW

**Comment by joshuaj194** [comment]
Deploy API changes are deployed to this app - sbx-unlink-replica-deploy-api - in the `sbx-apis` environment - https://app.aptible.com/apps/80850/services

App UI changes are deployed to this sandbox app - app-ui-sbx-unlink-replica - in the `sbx-unlink-replica` environment - https://app.aptible.com/apps/80923/services

File: src/ui/pages/database-detail-settings.tsx [hunk 1]
```diff
@@ -387,6 +388,69 @@ const DatabaseNameChange = ({ database }: DbProps) => {
...
+
+  return (
+    <div>
+      {database.initializeFrom && (
+        <form onSubmit={onSubmit} className="flex flex-col gap-4 mt-4">
+          <FormGroup
+            label="Unlink Replica from Source"
+            description={
+              <p>
+                If <strong>{database.handle}</strong> has been promoted to a
```
Comment (michaelwang13) [inline src/ui/pages/database-detail-settings.tsx:410]: I think this text needs updating so it's clear that this does not stop replication, and is only a metadata change.

**Review by neurosnap — COMMENTED** [review]
nice job on the tests!

File: src/ui/pages/database-detail-settings.tsx (hunk 1 above)
Comment (joshuaj194) [reply src/ui/pages/database-detail-settings.tsx]: updated the wording @michaelwang13!

File: src/ui/pages/database-detail-settings.tsx [hunk 2]
```diff
@@ -387,6 +388,63 @@ const DatabaseNameChange = ({ database }: DbProps) => {
...
+              className="w-70"
+              type="submit"
+            >
+              {/* <IconTrash color="#FFF" className="mr-2" /> */}
+              Unlink Replica from Source
+            </ButtonDestroy>
+          </Group>
+        </form>
+      )}
+      {<BannerMessages {...loader} />}
```
Comment (neurosnap) [inline src/ui/pages/database-detail-settings.tsx:442]: You don't need the curly brackets in this case

**Review by neurosnap — APPROVED** [review]

**Pull request merged**